### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by mahdanoura**

### CODE REVIEW

**Comment by mahdanoura** [comment]
@lu-zero could you please run the steps in the README and check whether you can run the code as I described it. Thank you.

File: README.md [hunk 1]
```diff
@@ -1,50 +1,63 @@
...
-## Prerequisites
+Step 1: Generate WoT resources using WOTIS
+
+Step 2: Generate the final WoT TD specification document using the generated WoT resources along with a static ```index.html```
+
+<img title="WoT Toolchain Overview" src="images/toolchain.svg">
 
-The [uv](https://docs.astral.sh/uv/) package manager.
+## Prerequisites
+* Python 3.11 or greater. [Download and install Python.](https://www.python.org/downloads/)
```
Comment (lu-zero) [inline README.md:22]: uv takes care of python, no need to install it separately.

File: README.md [hunk 2]
```diff
@@ -1,50 +1,63 @@
...
+```bash
 git clone https://github.com/w3c/wot-thing-description-toolchain-tmp.git
 cd wot-thing-description-toolchain-tmp
 ```
-2. Run the script using `uv`
-```
-uv run main.py -h
+2. Install the package
+```bash
+uv install
```
Comment (lu-zero) [inline README.md:33]: `uv run wotis` is all you need.

File: README.md [hunk 3]
```diff
@@ -1,50 +1,63 @@
...
+  --help                   Show this help message and exit.
 ```
+
 ## Examples
-Generate resources using the default schema and configuration:
-```
-uv run main.py
+Generate documentation using the default schema locally and serve it:
+```bash
+wotis generate-wot-resources -l -s
```
Comment (lu-zero) [inline README.md:58]:
```suggestion
wotis generate-wot-resources -l -s
```
```suggestion
wotis generate-wot-resources -d -s
```

File: README.md [hunk 4]
```diff
@@ -1,50 +1,63 @@
...
+2. Install the package
+```bash
+uv install
 ```
 
 ## Usage
-The main.py script supports various options:
+See the list of all WOTIS commands:
+```bash
+wotis --help
```
Comment (lu-zero) [inline README.md:39]:
```suggestion
uv run wotis --help
```

File: README.md [hunk 5]
```diff
@@ -1,50 +1,63 @@
...
 1. Clone the repository and navigate to the project directory:
-```
+```bash
 git clone https://github.com/w3c/wot-thing-description-toolchain-tmp.git
 cd wot-thing-description-toolchain-tmp
 ```
-2. Run the script using `uv`
-```
-uv run main.py -h
+2. Install the package
```
Comment (lu-zero) [inline README.md:31]:
```suggestion
2. Install the package or run it using `uv run wotis`
```

File: README.md (hunk 4 above)
Comment (mahdanoura) [reply README.md:39]: I suggest we keep the `wotis --help`, because the project installs it as a standalone command, making it simpler to use.

**Comment by lu-zero** [comment]
We can land this and iterate over :)

**Comment by lu-zero** [comment]
Meanwhile I fixed the CI, linkml does not support python 3.13 yet.

File: README.md [hunk 6]
```diff
@@ -1,50 +1,63 @@
 
-# Temporary WoT TD Tooling
+# WoTIS - Web of Things Integrated Schemas
 
-The aim of this repository is to simplify the current WoT TD tooling for generating the specification. WoT TD Tooling is a python project to automate the generation of
- 1) WoT resources: [SHACL Shapes](https://www.w3.org/TR/shacl/), [JSON Schema](https://json-schema.org/specification), [JSON-LD context](https://www.w3.org/TR/json-ld11/), and [RDF](https://www.w3.org/TR/rdf11-concepts/)
+The aim of this repository is to simplify the current tooling required for generating the WoT Thing Description (TD) specification and related resources. 
+WoTIS toolchain is a python-based project designed to automate the generation of
+ 1) WoT resources: [SHACL Shapes](https://www.w3.org/TR/shacl/), [JSON Schema](https://json-schema.org/specification), [JSON-LD context](https://www.w3.org/TR/json-ld11/), [RDF](https://www.w3.org/TR/rdf11-concepts/) and visual figures
```
Comment (TallTed) [inline README.md:6]:
```suggestion
 1) WoT resources: [SHACL Shapes](https://www.w3.org/TR/shacl/), [JSON Schema](https://json-schema.org/specification), [JSON-LD context](https://www.w3.org/TR/json-ld11/), [RDF](https://www.w3.org/TR/rdf11-concepts/), and visual figures
```

**Review by TallTed — CHANGES_REQUESTED** [review]
add human-facing comma for clarity

**Pull request merged**